### ISSUE TITLE
[lines-between-class-members] Fails if file contains no class

### ISSUE THREAD

**Issue by Wyctus** [eslint/eslint#14171]
* **ESLint Version: 7.20.0**
* **Node Version: 15.9.0**
* **npm Version: 7.5.3**
* **Operating System: Manjaro** 

**What parser (default, `@babel/eslint-parser`, `@typescript-eslint/parser`, etc.) are you using?**

`@typescript-eslint/parser`

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js
{
  root: true,
  parser: "@typescript-eslint/parser",
  parserOptions: {
    tsconfigRootDir: __dirname,
    project: ["./tsconfig.json"],
  },
  plugins: ["@typescript-eslint", "eslint-plugin-tsdoc"],
  extends: [
    "plugin:@typescript-eslint/recommended-requiring-type-checking",
    "airbnb-typescript",
    "prettier",
  ],
  rules: {
    "tsdoc/syntax": "warn",
    "no-console": "off",
    "lines-between-class-members": {
      "exceptAfterSingleLine": true
    }
  },
}
```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

If I only have files containing classes then with the `lines-between-class-members` rule everything works fine. But as soon as I create a file which contains no classes then this rule fails. 

I simply run ESLint by: 

```bash
eslint .
```

**What did you expect to happen?**
Probably it shouldn't deal with files not containing classes.

**What actually happened? Please copy-paste the actual, raw output from ESLint.**
Throws an error if the file contains no class.

```
Oops! Something went wrong! :(

ESLint: 7.20.0

Error: .eslintrc.js:
        Configuration for rule "lines-between-class-members" is invalid:
        Severity should be one of the following: 0 = off, 1 = warn, 2 = error (you passed '{ exceptAfterSingleLine: true }').

    at ConfigValidator.validateRuleOptions (/home/wyctus/d/tools/converter/node_modules/@eslint/eslintrc/lib/shared/config-validator.js:142:23)
    at /home/wyctus/d/tools/converter/node_modules/@eslint/eslintrc/lib/shared/config-validator.js:197:18
    at Array.forEach (<anonymous>)
    at ConfigValidator.validateRules (/home/wyctus/d/tools/converter/node_modules/@eslint/eslintrc/lib/shared/config-validator.js:194:34)
    at ConfigValidator.validateConfigArray (/home/wyctus/d/tools/converter/node_modules/@eslint/eslintrc/lib/shared/config-validator.js:320:18)
    at CascadingConfigArrayFactory._finalizeConfigArray (/home/wyctus/d/tools/converter/node_modules/@eslint/eslintrc/lib/cascading-config-array-factory.js:493:23)
    at CascadingConfigArrayFactory.getConfigArrayForFile (/home/wyctus/d/tools/converter/node_modules/@eslint/eslintrc/lib/cascading-config-array-factory.js:299:21)
    at FileEnumerator._iterateFilesRecursive (/home/wyctus/d/tools/converter/node_modules/eslint/lib/cli-engine/file-enumerator.js:440:49)
    at _iterateFilesRecursive.next (<anonymous>)
    at FileEnumerator.iterateFiles (/home/wyctus/d/tools/converter/node_modules/eslint/lib/cli-engine/file-enumerator.js:296:49)
```

**Comment by mdjermanovic** [comment]
Hi @Wyctus, thanks for the issue! 

```
Error: .eslintrc.js:
        Configuration for rule "lines-between-class-members" is invalid:
        Severity should be one of the following: 0 = off, 1 = warn, 2 = error (you passed '{ exceptAfterSingleLine: true }').
```

The error you're getting is due to a misconfiguration in your config file.

This:

```js
"lines-between-class-members": {
      "exceptAfterSingleLine": true
    }
```

Should be:

```js
"lines-between-class-members": ["error", "always", {
      "exceptAfterSingleLine": true
    }]
```

**Comment by mdjermanovic** [comment]
Also, since you are using `@typescript-eslint/parser`, you should probably enable [`@typescript-eslint/lines-between-class-members`](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/lines-between-class-members.md) rule instead of the core `lines-between-class-members` rule.
